fee_calculator/apps/calculator/management/commands/lgfs_2022_uplift.py

```diff
@@ -0,0 +1,107 @@
+from django.core.management import BaseCommand
+
+from calculator.models import (Scheme, Price)
+
+import csv
+
+
+def csv_to_json(csvFilePath):
+    jsonArray = []
+
+    with open(csvFilePath, encoding='utf-8-sig') as csvf:
+        csvReader = csv.DictReader(csvf)
+        for row in csvReader:
+            jsonArray.append(row)
+    return jsonArray
+
+
+base_path = 'fee_calculator/apps/calculator/management/commands/lgfs_2022_data'
```

Comment:
Could use a constants file and import that for base_path and help variables to make it easier to identify and edit them  (instead of having to comb) through the code, but more of a preference than anything